themes/default/content/what-is/amazon-dynamodb-vs-google-cloud-bigtable.md

```diff
@@ -0,0 +1,74 @@
+      - tableau
+      - atlassian
+      - fauna
+      - ware2go
+    - items:
+      - mindbody
+      - sourcegraph
+      - fenergo
+      - skai
+      - lemonade
+    - items:
+      - clearsale
+      - angellist
+      - webflow
+      - supabase
+      - ro
+---
+
+## Biggest Similarities and Differences Between Google Cloud Bigtable and AWS DynamoDB
+
+Both Google Cloud Bigtable and DynamoDB are [NoSQL]({{< relref "/learn/glossary/nosql" >}}), or non-relational, databases. This concept means their data is stored in some format other than two-dimensional tables. Some commonly-used formats for NoSQL databases in general are documents, key-value pairs, graphs, and columns, each with different strengths and tradeoffs.
+
+Google Cloud Bigtable and AWS DynamoDB are both highly-available, scalable, globally distributed and fully-managed serverless NoSQL databases. Both can function as a key-value store, however DynamoDB additionally supports a document model and Bigtable additionally supports a wide-column store. Both offer two consistency levels: eventual consistency, and immediate consistency.
+
+## Which NoSQL Database is better? Bigtable vs DynamoDB
+
+Choosing a database for your application requires careful consideration of your current and future needs. There are advantages and disadvantages to both Bitable and DynamoDB, but they differ enough that one will have a clear advantage for your specific application.
+
+### Pros and Cons of Bigtable
+
+Bigtable excels at handling massive quantities of data and providing very low latency for both read and write actions on that data. It is also possible to dynamically adjust throughput, by adding more nodes to the Bigtable cluster for times of particularly high demand, then deleting those nodes once normal activity resumes to keep costs down. BigTable also supports SQL-like queries, through the use of BigQuery.
+
+However, unlike DynamoDB, Bigtable does not support auto-scaling without first defining metrics to monitor and scale in response to. It also sacrifices some availability in exchange for consistency, whereas DynamoDB has made the opposite decision. It is exclusive to Google Cloud.
+
+### Pros and Cons of DynamoDB
+
+DynamoDB is extremely fast, with response times under 10ms, and also benefits from the entire suite of existing AWS services, including AWS IAM for fine-grained access control. It scales extremely well, with virtually unlimited storage available to you. It is a fully-managed service, meaning you are not responsible for managing any of the underlying infrastructure or upgrades.
+
+While it can be a very affordable option, predicting the costs accurately can be difficult regardless of which billing model you go with (on-demand or provisioned). Those who are used to the specificity of traditional SQL queries may also find the query options lacking, and scanning the database rather than querying it can result in a higher-than-expected bill. It is exclusive to AWS.
```

Comment:
```suggestion
While it can be a very affordable option, predicting the costs accurately can be difficult regardless of which billing model you go with (on-demand or provisioned). Those who are used to the specificity of traditional SQL queries may also find the query options lacking, and scanning the database rather than querying it can result in a higher-than-expected bill. DynamoDB is exclusive to AWS.
```